src/components/Calculator.js

```diff
@@ -1,129 +1,115 @@
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '7'))}>7</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '8'))}>8</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '9'))}>9</button>
+            </td>
+            <td className="orange_cell" type="button">
+              <button className="orange_cell no_border" type="submit" onClick={() => setMyState((myState) => calculate(myState, 'x'))}>x</button>
+            </td>
+          </tr>
+          <tr>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '4'))}>4</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '5'))}>5</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '6'))}>6</button>
+            </td>
+            <td className="orange_cell" type="button">
+              <button className="orange_cell no_border" type="submit" onClick={() => setMyState((myState) => calculate(myState, '-'))}>-</button>
+            </td>
+          </tr>
+          <tr>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '1'))}>1</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '2'))}>2</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '3'))}>3</button>
+            </td>
+            <td className="orange_cell" type="button">
+              <button className="orange_cell no_border" type="submit" onClick={() => setMyState((myState) => calculate(myState, '+'))}>+</button>
+            </td>
+          </tr>
+          <tr>
+            <td className="gray_cell" type="button" colSpan="2">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '0'))}>0</button>
+            </td>
+            <td className="gray_cell" type="button">
+              <button type="submit" onClick={() => setMyState((myState) => calculate(myState, '.'))}>.</button>
+            </td>
+            <td className="orange_cell" type="button">
+              <button className="orange_cell no_border" type="submit" onClick={() => setMyState((myState) => calculate(myState, '='))}>=</button>
```

Comment:
- [ ] Please double-check on the "=" button because it seems to be not showing the result of an operation! 😄

Reply:
@erezfree29 Please let's work on the issue mentioned by the previous reviewer.